central/image/datastore/store/postgres/store.go

```diff
@@ -389,10 +389,11 @@ func copyFromImageCves(ctx context.Context, tx *postgres.Tx, iTime time.Time, ob
 			obj.GetImpactScore(),
 			obj.GetSnoozed(),
 			protocompat.NilOrTime(obj.GetSnoozeExpiry()),
-			obj.GetCveBaseInfo().GetEpss().GetEpssProbability(),
 			serialized,
-		})
+		}
 
+		row = append(row, pgutils.EpssFloatOrNil(obj.GetCveBaseInfo().GetEpss()))
+		inputRows = append(inputRows, row)
```

Comment:
So hold on a minute.  Are you wanting the probability to be nil if the parent object is nil?  If so I retract my suggestion to put it in utils.  The reason being is that should be for things like NilOrFloat, just like NilOrTime.  So if we are just trying to avoid setting the 0 value when the outer object is nil, we should probably put it back the way you had it.  Sorry.